go.mod

```diff
@@ -2,9 +2,9 @@ module github.com/aws/amazon-cloudwatch-agent
 
 go 1.13
 
-replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20201015165757-4470de2d306b
+replace github.com/influxdata/telegraf => github.com/aws/telegraf v0.10.2-0.20220509174810-d4f1bd11c120
```

Comment:
Updating the aws sdk causes golang.org/x/net to update this we need to update telegraf.

Reply:
Do we need to update everything in this PR? should we pin to a specific version of the AWS SDK until the telegraf upgrade is complete?

Reply:
Agree. Upgrade with `aws-sdk` is exceptional since it does not directly involve with our main packages and only involves in cleaning `integration-test` resources.

Reply:
@SaxyPandaBear  Right. Then what we could do is create a separate branch from `upgrade-telegraf` and ensure it works from there too to avoid any unnecessary regression. For now, we will hold on merging until this work on `upgrade-telegraf`.